internal/xds/rbac/rbac_engine_test.go

```diff
@@ -1831,23 +1883,37 @@ func (builder *TestAuditLoggerCustomConfigBuilder) Build(config audit.LoggerConf
 }
 
 func (builder *TestAuditLoggerCustomConfigBuilder) Name() string {
-	return typedURLPrefix + "TestAuditLoggerCustomConfig"
+	return "TestAuditLoggerCustomConfig"
 }
 
-// This is used when converting a custom config from raw JSON to a TypedStruct
-// The TypeURL of the TypeStruct will be "grpc.authz.audit_logging/<name>"
-const typedURLPrefix = "grpc.authz.audit_logging/"
-
 // Builds custom configs for audit logger RBAC protos
 func anyPbHelper(t *testing.T, in map[string]interface{}, name string) *anypb.Any {
 	t.Helper()
 	pb, err := structpb.NewStruct(in)
-	typedURL := ""
-	if name != "" {
-		typedURL = typedURLPrefix + name
+	// typedURL := ""
+	// if name != "" {
+	// 	typedURL = typedURLPrefix + name
+	// }
+	typedStruct := &v1xdsudpatypepb.TypedStruct{
+		TypeUrl: typedURLPrefix + name,
+		Value:   pb,
 	}
-	typedStruct := &v1typepb.TypedStruct{
-		TypeUrl: typedURL,
+	if err != nil {
+		t.Fatal(err)
+	}
+	customConfig, err := anypb.New(typedStruct)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return customConfig
+}
+
+// Builds custom configs for audit logger RBAC protos
+func anyPbHelperXds(t *testing.T, in map[string]interface{}, name string) *anypb.Any {
```

Comment:
nit: The name is getting confusing. How about naming this `createXDSTypedStruct` and the one above `createUDPATypedStruct` or something similar?

Reply:
I like that more, changing